client_app/app/controllers/users_controller.rb

```diff
@@ -0,0 +1,326 @@
+
+	# require 'net/http'
+	# require 'uri'
+	# API_URL = "http://localhost:4000"
+
+
+
+
+	## ------- To debug the controller ----------##
+
+
+	#  To print the value into console
+	#    puts @user.inspect
+	#
+	#
+	#  To output the responce and search the data
+	# debugger
+
+
+	# => puts '@@current_user'
+		# puts @@current_user
+		# puts '@current_user'
+		# puts @current_user
+
+class UsersController < ApplicationController
+	 before_action :set_user, only: [:edit, :update, :destroy, :show ]
+	 before_action :authanticate_user, only: [:edit, :update, :destroy, :show]
+	
+	def index
+		res = Net::HTTP.get_response(UriCreaterServices.create_uri("/users"))
```

Comment:
Move the logic to `UriCreaterServices` so you need to call something like `UriCreaterServices.get_users`